cadl-tests/src/main/java/com/cadl/specialheaders/SpecialHeadersAsyncClient.java

```diff
@@ -169,17 +169,6 @@ public Mono<Response<BinaryData>> postWithResponse(String name, BinaryData body,
     /**
      * Send a LRO request with header Repeatability-Request-ID and Repeatability-First-Sent.
      *
-     * <p><strong>Header Parameters</strong>
-     *
-     * <table border="1">
-     *     <caption>Header Parameters</caption>
-     *     <tr><th>Name</th><th>Type</th><th>Required</th><th>Description</th></tr>
-     *     <tr><td>repeatability-request-id</td><td>String</td><td>No</td><td>Repeatability request ID header</td></tr>
-     *     <tr><td>repeatability-first-sent</td><td>String</td><td>No</td><td>Repeatability first sent header as HTTP-date</td></tr>
```

Comment:
Please log an issue to investigate.

Reply:
issue created: https://github.com/Azure/autorest.java/issues/1933